deploy/blobfuse-proxy.yaml

```diff
@@ -0,0 +1,58 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-blobfuse-proxy
```

Comment:
I think this proxy could be used in e2e test, while for production, we suggest install this proxy as a standalone process

Reply:
sure. Added a comment in the file explaining the same.